runtime/interpreter/value_type.go

```diff
@@ -172,6 +174,67 @@ func (v TypeValue) GetMember(interpreter *Interpreter, _ LocationRange, name str
 		}
 
 		return AsBoolValue(elaboration.IsRecovered)
+
+	case sema.MetaTypeAddressFieldName:
+		staticType := v.Type
+		if staticType == nil {
+			return Nil
+		}
+
+		location, _, err := common.DecodeTypeID(interpreter, string(staticType.ID()))
+		if err != nil || location == nil {
+			return Nil
+		}
```

Comment:
rather than decoding the type-ID, it would be efficient to check whether `v.Type` is either the `CompositeStaticType`/`InterfaceStaticType` and return the locations from the type itself (and nil, otherwise)

Reply:
Good idea! Changed it here and for the other field below in d90d44960c54be16ec68785ad5946f042ab8df3e